app/reports_generation/csv_chart_generator.py

```diff
@@ -1,7 +1,8 @@
 import datetime
 from pathlib import Path
 
-from scripts import config_provider, csv_aggregator, chart_generator, summary_aggregator, results_archivator
+from reports_generation.scripts import (config_provider, csv_aggregator, chart_generator,
```

Comment:
Could you pls make a script to work from the `app/reports_generation` folder as this is described in the documentation.